.github/workflows/build_test.yml

```diff
@@ -9,14 +9,7 @@ on:
       - '.github/workflows/changelog_test.yml'
       - 'docker/**'
       - 'CHANGELOG.rst'
-  push:
```

Comment:
haven't we traditionally done a built & test on merge (aka push)?  I'm on the fence about whether or not it is necessary, but removing this is a departure from our CircleCI testing (I think)